### ISSUE TITLE
Tooltip inside DropdownMenuEntry causes infinite exceptions and app freeze

### ISSUE THREAD

**Issue by dabbinavo** [flutter/flutter#135742]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

1. Run the provided code sample in debug mode
2. Hover the mouse over one of the two question mark icons
3.a See the app freeze
3.b See infinite Exceptions in the debug console

### Expected results

Not freezing or throwing exceptions

### Actual results

Freezes and throws infinite Exceptions

### Code sample

<details><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      home: Scaffold(
        body: Center(
          child: DropdownMenu(
            dropdownMenuEntries: [
              DropdownMenuEntry(
                value: 1,
                label: 'A label',
                trailingIcon: IconButton(
                  onPressed: null,
                  icon: Icon(Icons.question_mark),
                  tooltip: 'A tooltip',
                ),
              ),
              DropdownMenuEntry(
                value: 2,
                label: 'B label',
                trailingIcon: Tooltip(
                  message: 'B tooltip',
                  child: Icon(Icons.question_mark),
                ),
              ),
            ],
          ),
        ),
      ),
    );
  }
}

```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

App throws infinite exceptions and freezes when hovering the question marks with the mouse

![image](https://github.com/flutter/flutter/assets/14122508/d7679d00-bb66-445e-b5b7-e97a7ba2cc56)

</details>

### Logs

<details><summary>Logs</summary>

```console
══╡ EXCEPTION CAUGHT BY SCHEDULER LIBRARY ╞═════════════════════════════════════════════════════════
The following assertion was thrown during a scheduler callback:
Cannot hit test a render box that has never been laid out.
The hitTest() method was called on this RenderBox: _RenderDeferredLayoutBox#3e2d5 NEEDS-LAYOUT NEEDS-PAINT:
  creator: _DeferredLayout ← _OverlayPortal ← OverlayPortal ← Tooltip ← Padding ← Row ← _MenuItemLabel
    ← Align ← Padding ← IconTheme ← Builder ← Listener ← ⋯
  parentData: not positioned; offset=Offset(0.0, 0.0)
  constraints: MISSING
  size: MISSING
Unfortunately, this object's geometry is not known at this time, probably because it has never been
laid out. This means it cannot be accurately hit-tested.
If you are trying to perform a hit test during the layout phase itself, make sure you only hit test
nodes that have completed layout (e.g. the node's children, after their layout() method has been
called).

When the exception was thrown, this was the stack:
#0      RenderBox.hitTest.<anonymous closure> (package:flutter/src/rendering/box.dart:2442:11)
box.dart:2442
#1      RenderBox.hitTest (package:flutter/src/rendering/box.dart:2473:6)
box.dart:2473
#2      _RenderTheaterMixin.hitTestChildren.childHitTest (package:flutter/src/widgets/overlay.dart:856:81)
overlay.dart:856
#3      BoxHitTestResult.addWithPaintOffset (package:flutter/src/rendering/box.dart:800:31)
box.dart:800
#4      _RenderTheaterMixin.hitTestChildren (package:flutter/src/widgets/overlay.dart:857:22)
overlay.dart:857
#5      RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#6      RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#7      RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#8      RenderAbsorbPointer.hitTest (package:flutter/src/rendering/proxy_box.dart:3856:17)
proxy_box.dart:3856
#9      RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#10     RenderProxyBoxWithHitTestBehavior.hitTest (package:flutter/src/rendering/proxy_box.dart:172:19)
proxy_box.dart:172
#11     RenderBoxContainerDefaultsMixin.defaultHitTestChildren.<anonymous closure> (package:flutter/src/rendering/box.dart:2854:25)
box.dart:2854
#12     BoxHitTestResult.addWithPaintOffset (package:flutter/src/rendering/box.dart:800:31)
box.dart:800
#13     RenderBoxContainerDefaultsMixin.defaultHitTestChildren (package:flutter/src/rendering/box.dart:2849:33)
box.dart:2849
#14     RenderFlex.hitTestChildren (package:flutter/src/rendering/flex.dart:1034:12)
flex.dart:1034
#15     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#16     RenderBoxContainerDefaultsMixin.defaultHitTestChildren.<anonymous closure> (package:flutter/src/rendering/box.dart:2854:25)
box.dart:2854
#17     BoxHitTestResult.addWithPaintOffset (package:flutter/src/rendering/box.dart:800:31)
box.dart:800
#18     RenderBoxContainerDefaultsMixin.defaultHitTestChildren (package:flutter/src/rendering/box.dart:2849:33)
box.dart:2849
#19     RenderStack.hitTestChildren (package:flutter/src/rendering/stack.dart:630:12)
stack.dart:630
#20     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#21     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#22     RenderProxyBoxWithHitTestBehavior.hitTest (package:flutter/src/rendering/proxy_box.dart:172:19)
proxy_box.dart:172
#23     RenderBoxContainerDefaultsMixin.defaultHitTestChildren.<anonymous closure> (package:flutter/src/rendering/box.dart:2854:25)
box.dart:2854
#24     BoxHitTestResult.addWithPaintOffset (package:flutter/src/rendering/box.dart:800:31)
box.dart:800
#25     RenderBoxContainerDefaultsMixin.defaultHitTestChildren (package:flutter/src/rendering/box.dart:2849:33)
box.dart:2849
#26     RenderCustomMultiChildLayoutBox.hitTestChildren (package:flutter/src/rendering/custom_layout.dart:415:12)
custom_layout.dart:415
#27     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#28     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#29     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#30     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#31     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#32     RenderPhysicalModel.hitTest (package:flutter/src/rendering/proxy_box.dart:1990:18)
proxy_box.dart:1990
#33     _RenderLayoutBuilder.hitTestChildren (package:flutter/src/widgets/layout_builder.dart:332:19)
layout_builder.dart:332
#34     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#35     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#36     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#37     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#38     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#39     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#40     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#41     RenderIgnorePointer.hitTest (package:flutter/src/rendering/proxy_box.dart:3609:31)
proxy_box.dart:3609
#42     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#43     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#44     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#45     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#46     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#47     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#48     RenderOffstage.hitTest (package:flutter/src/rendering/proxy_box.dart:3738:31)
proxy_box.dart:3738
#49     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#50     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#51     _RenderTheaterMixin.hitTestChildren.childHitTest (package:flutter/src/widgets/overlay.dart:856:81)
overlay.dart:856
#52     BoxHitTestResult.addWithPaintOffset (package:flutter/src/rendering/box.dart:800:31)
box.dart:800
#53     _RenderTheaterMixin.hitTestChildren (package:flutter/src/widgets/overlay.dart:857:22)
overlay.dart:857
#54     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#55     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#56     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#57     RenderAbsorbPointer.hitTest (package:flutter/src/rendering/proxy_box.dart:3856:17)
proxy_box.dart:3856
#58     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#59     RenderProxyBoxWithHitTestBehavior.hitTest (package:flutter/src/rendering/proxy_box.dart:172:19)
proxy_box.dart:172
#60     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#61     RenderCustomPaint.hitTestChildren (package:flutter/src/rendering/custom_paint.dart:544:18)
custom_paint.dart:544
#62     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#63     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#64     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#65     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#66     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#67     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#68     RenderTapRegionSurface.hitTest (package:flutter/src/widgets/tap_region.dart:213:28)
tap_region.dart:213
#69     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#70     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#71     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#72     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#73     RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:117:19)
proxy_box.dart:117
#74     RenderBox.hitTest (package:flutter/src/rendering/box.dart:2475:11)
box.dart:2475
#75     RenderView.hitTest (package:flutter/src/rendering/view.dart:194:14)
view.dart:194
#76     RendererBinding.hitTestInView (package:flutter/src/rendering/binding.dart:534:16)
binding.dart:534
#77     RendererBinding.initMouseTracker.<anonymous closure> (package:flutter/src/rendering/binding.dart:319:7)
binding.dart:319
#78     MouseTracker._findAnnotations (package:flutter/src/rendering/mouse_tracker.dart:261:60)
mouse_tracker.dart:261
#79     MouseTracker.updateAllDevices.<anonymous closure> (package:flutter/src/rendering/mouse_tracker.dart:375:80)
mouse_tracker.dart:375
#80     MouseTracker._deviceUpdatePhase (package:flutter/src/rendering/mouse_tracker.dart:210:9)
mouse_tracker.dart:210
#81     MouseTracker.updateAllDevices (package:flutter/src/rendering/mouse_tracker.dart:372:5)
mouse_tracker.dart:372
#82     RendererBinding._scheduleMouseTrackerUpdate.<anonymous closure> (package:flutter/src/rendering/binding.dart:377:22)
binding.dart:377
#83     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1297:15)
binding.dart:1297
#84     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1236:9)
binding.dart:1236
#85     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1085:5)
binding.dart:1085
#86     _invoke (dart:ui/hooks.dart:170:13)
hooks.dart:170
#87     PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:401:5)
platform_dispatcher.dart:401
#88     _drawFrame (dart:ui/hooks.dart:140:31)
hooks.dart:140
════════════════════════════════════════════════════════════════════════════════════════════════════
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
Another exception was thrown: Cannot hit test a render box that has never been laid out.
...
```

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[√] Flutter (Channel stable, 3.13.6, on Microsoft Windows [Version 10.0.19044.3448], locale de-DE)
    • Flutter version 3.13.6 on channel stable at C:\Users\fischerauer\.flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ead455963c (2 days ago), 2023-09-26 18:28:17 -0700
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Users\fischerauer\AppData\Local\Android\sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Portable\jdk-20.0.2\bin\java
    • Java version Java(TM) SE Runtime Environment (build 20.0.2+9-78)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Build Tools 2022 17.6.1)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools
    • Visual Studio Build Tools 2022 version 17.6.33717.318
    • Windows 10 SDK version 10.0.22000.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] VS Code (version 1.82.2)
    • VS Code at C:\Users\fischerauer\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.72.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.3448]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.67
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 117.0.2045.31

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

</details>

**Comment by dam-ease** [comment]
Hi @dabbinavo. Thanks for filing this. I can reproduce this on the latest `master` and `stable` channels following the steps described above.

<details><summary>stable, master flutter doctor -v</summary>
<p>

```
[✓] Flutter (Channel stable, 3.13.6, on macOS 13.0 22A380 darwin-arm64, locale
    en-NG)
    • Flutter version 3.13.6 on channel stable at
      /Users/damilolaalimi/sdks/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ead455963c (2 days ago), 2023-09-26 18:28:17 -0700
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/damilolaalimi/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/damilolaalimi/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      17.0.6+0-17.0.6b802.4-9586694)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.12.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      17.0.6+0-17.0.6b802.4-9586694)

[!] Android Studio (version unknown)
    • Android Studio at /Users/damilolaalimi/Downloads/Android Studio
      Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to determine Android Studio version.
    • Java version OpenJDK Runtime Environment (build
      17.0.7+0-17.0.7b1000.6-10550314)

[✓] VS Code (version 1.82.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.50.0

[✓] Connected device (5 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554                        •
      android-arm64  • Android 14 (API 34) (emulator)
    • iPhone 14 Pro (mobile)      • 6548B0D1-7674-4306-9852-198C465F1104 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)
    • iPhone 14 Pro Max (mobile)  • BB55E997-7F31-462D-B3B1-6B177D9B40C7 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)
    • macOS (desktop)             • macos                                •
      darwin-arm64   • macOS 13.0 22A380 darwin-arm64
    • Chrome (web)                • chrome                               •
      web-javascript • Google Chrome 117.0.5938.132

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
``` 
```
[✓] Flutter (Channel master, 3.15.0-11.0.pre.44, on macOS 13.0 22A380 darwin-arm64, locale en-NG)
    • Flutter version 3.15.0-11.0.pre.44 on channel master at /Users/damilolaalimi/fvm/versions/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 938bb217fb (4 hours ago), 2023-09-28 21:11:31 -0700
    • Engine revision 2052515c44
    • Dart version 3.2.0 (build 3.2.0-210.0.dev)
    • DevTools version 2.28.1

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/damilolaalimi/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/damilolaalimi/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.12.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b802.4-9586694)

[!] Android Studio (version unknown)
    • Android Studio at /Users/damilolaalimi/Downloads/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to determine Android Studio version.
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)

[✓] VS Code (version 1.82.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.50.0

[✓] Connected device (5 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554                        • android-arm64  • Android 14 (API 34) (emulator)
    • iPhone 14 Pro (mobile)      • 6548B0D1-7674-4306-9852-198C465F1104 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)
    • iPhone 14 Pro Max (mobile)  • BB55E997-7F31-462D-B3B1-6B177D9B40C7 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-16-4 (simulator)
    • macOS (desktop)             • macos                                • darwin-arm64   • macOS 13.0 22A380 darwin-arm64
    • Chrome (web)                • chrome                               • web-javascript • Google Chrome 117.0.5938.132

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
``` 

</p>
</details>

**Comment by TahaTesser** [comment]
This seems to be fixed on the master branch including `3.15.0-13.0.pre.2`

https://github.com/flutter/flutter/assets/48603081/5d73bcd1-edcd-486a-98fb-2db7acaf4e84
